test/spec/TokenManager/browser.ts

```diff
@@ -547,85 +448,6 @@ describe('TokenManager (browser)', function() {
         });
       });
     });
-
-    it('automatically renews a token early when local clock offset is considered', function() {
-      var expiresAt = tokens.standardAccessTokenParsed.expiresAt;
-      return oauthUtil.setupFrame({
-        authClient: setupSync({
-          // local clock offset: 10 seconds behind the server
-          localClockOffset: 10000,
-          tokenManager: {
-            autoRenew: true
-          }
-        }),
-        autoRenew: true,
-        fastForwardToTime: true,
-        autoRenewTokenKey: 'test-accessToken',
-        tokenTypesTobeRenewed: ['access_token'],
-        time: expiresAt - 10, // set local time to 10 seconds until expiration
-        tokenManagerAddKeys,
-        postMessageSrc,
-        postMessageResp
-      })
-      .then(() => {
-        oauthUtil.expectTokenStorageToEqual(localStorage, {
-          'test-accessToken': { 
-            ...tokens.standardAccessTokenParsed, 
-            expiresAt: expiresAt - 10 + 3600
-          }
-        });
-      });
-    });
-
-    it('renews a token early when "expireEarlySeconds" option is considered', function() {
-      var expiresAt = tokens.standardAccessTokenParsed.expiresAt;
-      return oauthUtil.setupFrame({
-        authClient: setupSync({
-          tokenManager: {
-            autoRenew: true,
-            expireEarlySeconds: 10
-          }
-        }),
-        autoRenew: true,
-        fastForwardToTime: true,
-        autoRenewTokenKey: 'test-accessToken',
-        tokenTypesTobeRenewed: ['access_token'],
-        time: expiresAt - 10, // set local time to 10 seconds until expiration
-        tokenManagerAddKeys,
-        postMessageSrc,
-        postMessageResp
-      })
-      .then(function() {
-        oauthUtil.expectTokenStorageToEqual(localStorage, {
-          'test-accessToken': { 
-            ...tokens.standardAccessTokenParsed, 
-            expiresAt: expiresAt - 10 + 3600
-          }
-        });
-      });
-    });
-
-    it('does not return the token after tokens were cleared before renew promise was resolved', function() {
```

Comment:
Removed due to this test does not make sense. As long as the renew process starts, renewed tokens should be added to storage, original storage should not matter.